recipes/gnulib/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, tools
+import os
+import re
+import shutil
+
+
+class GnuLibConanFile(ConanFile):
+    name = "gnulib"
+    description = "Gnulib is a central location for common GNU code, intended to be shared among GNU packages."
+    homepage =  "https://www.gnu.org/software/gnulib/"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "gnulib", "library", "gnu")
+    license = ("GPL-3.0-or-later", "LGPL-3.0-or-later", "Public domain")
```

Comment:
```suggestion
    license = ("GPL-3.0-or-later", "LGPL-3.0-or-later", "Unlicense")
```

The unlicense is a public domain license: https://spdx.org/licenses/Unlicense.html